.github/workflows/main.yml

```diff
@@ -15,15 +15,14 @@ jobs:
       fail-fast: false
       matrix:
         task: ["pytest", "docs", "pyright"]
-        python-version: ["3.9"]
 
     steps:
       - uses: actions/checkout@v2
 
       - name: Setup Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: "3.10.6"
```

Comment:
hmmm.  not 3.9, and not >=3.9,<3.12

Reply:
Yup, I changed the version everywhere to 3.10.6